modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/GetValueFunction.java

```diff
@@ -57,6 +62,16 @@ public void setDDMExpressionFieldAccessor(
 		_ddmExpressionFieldAccessor = ddmExpressionFieldAccessor;
 	}
 
+	private boolean _isFieldVisible(String field) {
+		GetFieldPropertyRequest.Builder builder =
+			GetFieldPropertyRequest.Builder.newBuilder(field, "visible");
+
+		GetFieldPropertyResponse getFieldPropertyResponse =
+			_ddmExpressionFieldAccessor.getFieldProperty(builder.build());
+
+		return GetterUtil.getBoolean(getFieldPropertyResponse.getValue(), true);
```

Comment:
@linolaoj, why did you choose to use `true` as the default value?

Reply:
Hi @natocesarrego , I noticed that the form fields only had the property visible if it was changed, so letting the default value be false would not return the values of the fields whose visibility didn't change.

Reply:
@linolaoj, from what I could understand of the issue, the ideal fix would be set the field's value to empty, but here I believe you're just returning blank as the result of the execution of the function `getValue`, am I right?

If so, I understand that it can lead to other issues once the Forms context will be in a different state, e.g. with the value inputted by the user and not blank, does it make sense to you?

Can we try to update the field's value and visibility according to the expected behavior described on [this comment](https://issues.liferay.com/browse/PTR-1582?focusedCommentId=2112369&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2112369)?

Thanks.

Reply:
Hi @natocesarrego , I'll look for a way to also update the value on the context. I'll send you a new pull.
Thank you.